### PR TITLE
Fix pagesize not highlighted

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/8813

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-app-files/src/components/AppBar/ViewOptions.vue [hunk 1]
```diff
@@ -59,10 +59,10 @@
         <li v-if="hasPagination" class="files-view-options-list-item">
           <oc-page-size
             v-if="!queryParamsLoading"
-            :selected="queryItemAsString(itemsPerPageCurrent)"
+            :selected="itemsPerPageCurrent"
```
Comment (JammingBen) [inline packages/web-app-files/src/components/AppBar/ViewOptions.vue:62]: That was actually fine:

```suggestion
            :selected="queryItemAsString(itemsPerPageCurrent)"
```

Comment (JanAckermann) [reply packages/web-app-files/src/components/AppBar/ViewOptions.vue:62]: wtf

**Review by JammingBen — APPROVED** [review]

File: packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue [hunk 2]
```diff
@@ -333,6 +326,7 @@ export default defineComponent({
         } as AppRoleAssignment)
         return
       }
+      console.log(this.editUser)
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Users/SideBar/EditPanel.vue:329]:
```suggestion
```

**Review by JammingBen — CHANGES_REQUESTED** [review]
There is also a login input in `LoginModal.vue` where this needs to be done.

**Review by JammingBen — APPROVED** [review]
Approving meanwhile so Jan can finish and merge it later.

**Pull request merged**